integration/kubeseal_test.go

```diff
@@ -209,3 +209,26 @@ var _ = Describe("kubeseal --fetch-cert", func() {
 			Should(Equal(certs))
 	})
 })
+
+var _ = Describe("kubeseal --version", func() {
+	var c corev1.CoreV1Interface
+	var input io.Reader
+	var output *bytes.Buffer
+	var args []string
+
+	BeforeEach(func() {
+		c = corev1.NewForConfigOrDie(clusterConfigOrDie())
+
+		args = append(args, "--version")
```

Comment:
I get bitten by this ginkgo misfeature all the time:  you should set `args` (not append) in the "base" `BeforeEach` (here!) to avoid accumulating `args` between otherwise independent tests.  If you've copied this from somewhere else, then it means I've made that mistake there too (again).

You won't see it in this specific example because you've only got a single `It`, but add another one and print out `args` ...

Reply:
Corrected it. I copied it from just above, there 4 or so others in the same file - should we correct them here?